src/renderer/components/loans/OpenLoan.tsx

```diff
@@ -0,0 +1,1786 @@
+
+            return price ? `${price} (${fee})` : fee
+          }
+        )
+      ),
+
+    [loanFeesRD, oPriceAssetInFee]
+  )
+
+  //calculating transaction time from chain & quote
+  const transactionTime: Time = useMemo(() => {
+    return calculateTransactionTime(sourceChain)
+  }, [sourceChain])
+
+  const oWalletAddress: O.Option<Address> = useMemo(() => {
+    return FP.pipe(
+      sequenceTOption(oSourceAssetWB),
+      O.map(([{ walletAddress }]) => walletAddress)
+    )
+  }, [oSourceAssetWB])
+
+  const [showDetails, setShowDetails] = useState<boolean>(true)
+
+  return (
+    <div className="flex w-full max-w-[500px] flex-col justify-between py-[60px]">
+      <div>
+        <div className="flex flex-col">
+          <div className="text-12 text-gray2 dark:border-gray1d dark:text-gray2d">
+            <div className="rounded text-warning0 dark:text-warning0d">
+              {intl.formatMessage({ id: 'common.featureUnderDevelopment' })}
+            </div>
+            <div className="flex pb-4"></div>
+          </div>
+          <AssetInput
+            className="w-full"
+            title={intl.formatMessage({ id: 'common.collateral' })}
+            amount={{ amount: amountToLoanMax1e8, asset: collateralAsset.asset }}
+            priceAmount={{ asset: priceAmountToLoanMax1e8.asset, amount: priceAmountToLoanMax1e8.baseAmount }}
+            assets={selectableAssets}
+            network={network}
+            onChangeAsset={setCollateralAsset}
+            onChange={setAmountToLoanMax1e8}
+            onBlur={reloadFeesHandler}
+            showError={minAmountError}
+            hasLedger={hasLedger}
+            useLedger={useLedger}
+            useLedgerHandler={onClickUseLedger}
+            extraContent={
+              <div className="flex flex-col">
+                <MaxBalanceButton
+                  className="ml-10px mt-5px"
+                  classNameButton="!text-gray2 dark:!text-gray2d"
+                  classNameIcon={
+                    // show warn icon if maxAmountToLoanMax1e8 <= 0
+                    maxAmountToLoanMax1e8.gt(zeroBaseAmountMax1e8)
+                      ? `text-gray2 dark:text-gray2d`
+                      : 'text-warning0 dark:text-warning0d'
+                  }
+                  size="medium"
+                  balance={{ amount: maxAmountToLoanMax1e8, asset: collateralAsset.asset }}
+                  maxDollarValue={priceAmountMax1e8}
+                  onClick={() => setAmountToLoanMax1e8(maxAmountToLoanMax1e8)}
+                  maxInfoText={maxBalanceInfoTxt}
+                />
+                {minAmountError && renderMinAmount}
+              </div>
+            }
+          />
+
+          <div className="w-full px-20px">{renderSlider}</div>
+          <div className="flex w-full flex-col items-center justify-center px-5">
+            <div className="flex pt-6 text-gray2 dark:text-gray2d">
+              {intl.formatMessage({ id: 'loan.detail.creditRatio' })}
```

Comment:
Please change to Loan Collateralisation 200% and remove the slider bar. 
The user cannot change the loan CR. Min and Max CR in mimir are currently set to 200% hence the CR is fixed to 200%